src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo;
+  onClose?: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo: { title, id, user, completed },
+  onClose = () => {},
+}) => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
You're initializing your state with a static value, which is fine in this case. However, be careful when initializing your state with props, as this can lead to bugs and inconsistencies in your app. Here are some resources for further reading: [stackoverflow](https://stackoverflow.com/a/50403930), [reactjs.org](https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state)